Sources/Moya/Endpoint.swift

```diff
@@ -99,7 +72,33 @@ extension Endpoint {
         request.httpMethod = method.rawValue
         request.allHTTPHeaderFields = httpHeaderFields
 
-        return try? parameterEncoding.encode(request, with: parameters)
+        switch task {
+        case .requestPlain, .uploadFile, .uploadMultipart, .downloadDestination:
+            return request
+
```

Comment:
This is a stylistic one: can we remove the empty lines in the code (separating cases, first line empty in a method body)? This is probably a lot nicer for many people (including you), but I don't think we want to change the code style right now, especially with so much code changes.